SRanipalExtTrackingModule/SRanipalExtTrackingModule.csproj.user

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="Current" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <ProjectView>ProjectFiles</ProjectView>
+  </PropertyGroup>
+  <PropertyGroup>
+    <ReferencePath>
+    </ReferencePath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <StartAction>Project</StartAction>
+    <StartProgram>C:\Users\lazlo\source\repos\VRCFaceTracking\VRCFaceTracking\bin\Release\VRCFaceTracking.exe</StartProgram>
```

Comment:
We might wanna remove this `StartProgram` block, or the file itself as it contains somewhat sensitive data and is autogenerated

Reply:
Should remove it since it doesn't affect building the project.

Reply:
Removed in https://github.com/benaclejames/VRCFaceTracking/pull/109/commits/1c56093db19ef05654b4880307512d69920fc441